### PR TITLE
Installation 4.0

### PR DESCRIPTION
**Pull request by BitOne**
**Description**

This PR fixes installation instruction for the 4.0.

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: install_pim/docker/installation_docker.rst [hunk 1]
```diff
@@ -159,13 +152,13 @@ If you want to execute only one job:
...
 
-*Xdebug* is deactivated by default. If you want to activate it, you can use the environment variable ``XDEBUG_ENABLED`` and set its value to 1.
+*Xdebug* is disabled by default. If you want to enable it, set the environment variable ``XDEBUG_ENABLED`` to `1`.
 
 Enable it on fpm service:
 
 .. code-block:: bash
 
-    $ XDEBUG_ENABLED=1 make up
+    $ XDEBUG_ENABLED=1 make dev
```
Comment (AymericPlanche) [inline install_pim/docker/installation_docker.rst]:
```suggestion
    $ XDEBUG_ENABLED=1 make pim-dev
```

File: install_pim/docker/installation_docker.rst [hunk 2]
```diff
@@ -1,151 +1,144 @@
...
-
-Of course, it does not delete the Akeneo application you cloned on your machine, only the Docker containers. However, it destroys the database and everything it contains.
-
-
-Install and run Akeneo
-----------------------
-
-Configure Akeneo
-****************
+	make dev
```
Comment (AymericPlanche) [inline install_pim/docker/installation_docker.rst]:
```suggestion
	make pim-dev
```

File: install_pim/docker/installation_docker.rst [hunk 3]
```diff
@@ -1,151 +1,144 @@
...
-Now, you can initialize Akeneo by running:
+Launching the PIM in prod mode
+------------------------------
 
 .. code-block:: bash
 
-   $ make pim-prod
-
-.. code-block:: bash
+   $ make prod
```
Comment (AymericPlanche) [inline install_pim/docker/installation_docker.rst]:
```suggestion
   $ make pim-prod
```

File: install_pim/docker/installation_docker.rst (hunk 1 above)
Comment (BitOne) [reply install_pim/docker/installation_docker.rst]: Nope, it's not the same Makefile. It does more than just pim-dev or pim-prod ;)

File: install_pim/docker/installation_docker.rst (hunk 2 above)
Comment (jmleroux) [inline install_pim/docker/installation_docker.rst]:
```suggestion
	make pim-dev
```

File: install_pim/docker/installation_docker.rst (hunk 3 above)
Comment (jmleroux) [inline install_pim/docker/installation_docker.rst]:
```suggestion
   $ make pim-prod
```

File: install_pim/docker/installation_docker.rst (hunk 1 above)
Comment (jmleroux) [inline install_pim/docker/installation_docker.rst]:
```suggestion
    $ XDEBUG_ENABLED=1 make pim-dev
```

File: install_pim/docker/installation_docker.rst [hunk 4]
```diff
@@ -1,151 +1,144 @@
...
 .. code-block:: bash
 
-   $ make up
-
-To stop the containers, run:
+    $ mkdir pim
+    $ tar -xvjf pim-enterprise-standard-v4.0.tar.gz -C pim
+    $ cd pim
+    $ docker run -u www-data -v $(pwd):/srv/pim -w /srv/pim --rm akeneo/pim-php-dev:4.0 \
+        php -d memory_limit=4G /usr/local/bin/composer install
```
Comment (jmleroux) [inline install_pim/docker/installation_docker.rst]: Why do we need to run the `composer install`? Is it not available via `make`?

File: install_pim/docker/installation_docker.rst (hunk 2 above)
Comment (BitOne) [reply install_pim/docker/installation_docker.rst]: Nope, this is make dev.

Comment (BitOne) [reply install_pim/docker/installation_docker.rst]: https://github.com/akeneo/pim-enterprise-dev/blob/master/std-build/Makefile#L63

File: install_pim/docker/installation_docker.rst (hunk 4 above)
Comment (BitOne) [reply install_pim/docker/installation_docker.rst]: No, because the needed Makefile is provided by composer post-install script.

**Pull request merged**